server/src/main/scala/org/broadinstitute/dsp/workbench/welder/server/ObjectService.scala

```diff
@@ -98,17 +99,18 @@ class ObjectService(
     for {
       context <- storageLinksAlg.findStorageLink(req.localObjectPath)
       current <- timer.clock.realTime(TimeUnit.MILLISECONDS)
+      gsPath = getGsPath(req.localObjectPath, context)
+      hashedLockedByCurrentUser <- hashString(lockedByString(gsPath.bucketName, config.ownerEmail))
```

Comment:
Does this include the gs:// or not? I don't care either way, but the details should probably go into the README or the API yaml docs so clients have something to reference to unpack this value.

Reply:
it doesn't include the `gs://` prefix. i'll document.

Reply:
Thanks. Also I misspoke by saying "unpack"; clients will need to generate the hash in the same way as Welder (but by design, the hash is not "unpackable").